x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_list_page/components/bulk_actions.tsx

```diff
@@ -121,7 +108,6 @@ export const AgentBulkActions: React.FunctionComponent<Props> = ({
         />
       ),
       icon: <EuiIcon type="tag" size="m" />,
-      disabled: !atLeastOneActiveAgentSelected,
```

Comment:
Way better without those checks, preventing these actions created a lot of issue!